Amazon.SQS/src/Extensions.SQS/Collector/SqsQueueMessage.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace Azure.Functions.Extensions.SQS.Collector
+{
+	public class SqsQueueMessage
```

Comment:
I would also add initial visibility delay property. I can see clients having a use for it.

Reply:
I've added a binding for the native amazon sqs sdk message so as to have access to all of the supported options. 
But I also left this class for clients who only care about the body they are pushing.